src/01-functions.js

```diff
@@ -24,7 +26,9 @@ function alwaysTrue() {}
  *  greet("Ian");
  *  //> "Hello, Ian."
  */
-function greet() {}
+function greet(name) {
+  return "Hello, " + name + ".";
```

Comment:
Can we do this with template literals?  Code looks VERY clean Aisha.  Great job.